extensions/processing/README.md

```diff
@@ -0,0 +1,64 @@
+# Processing Extension Specification
+
+- **Title: Processing**
+- **Identifier: proc**
+- **Field Name Prefix: proc**
+- **Scope: Item, Collection**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+Processing metadata is considered to be data that indicate from which processing chain a data originates and how the data itself has been produced. Overall, it helps to increase traceability and search among processing levels and multiple algorithm versions.
+
+Often, data items are the result of one or more waterfall processing pipeline. Tracing information such as the processing facility , the algorithm version or the processing date helps in the data version management.
+
+This extension applies to STAC Items and STAC Collections. As these processing information are often closely bound to the collection level and therefore are shared across all items, it is recommended adding the fields to the corresponding STAC Collection.
+
+- [Examples](examples/): [Sentinel-1 Item](examples/sentinel1-item.json),  [Sentinel-2 Level 1C Collection](examples/sentinel2-collection.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Item and Collection fields
+
+For Items, the fields are placed in the properties. For Collections, the fields are placed in the summaries of the Collection.
+
+| Field Name                  | Type      | Description                                                  |
+| --------------------------- | --------- | ------------------------------------------------------------ |
+| proc:name         | string | The name of the processing of the data that is commonly used. For example, `GRD Post Processing` for "GRD" product of Sentinel-1 satellites. |
```

Comment:
What's the difference between proc:name and for example sar:product_type? Could that be somehow aligned?

Reply:
proc:name would be a label for the processing like an identifier but maybe a bit redundant if the `lineage` property is integrated. `sar:product_type`, which IMO should be a common metadata, is the family product identifier defined by the provider

Reply:
Yes, we discussed whether product_type should be in common metadata. The reason not to include it is that we want to define a list of "common" values like in the SAR extension, but that is a bit hard in common metadata. I guess we'll have follow-up discussions.

Replacing proc:name with proc:lineage makes sense, I think.